corehq/doctypemigrations/migrator.py

```diff
@@ -112,3 +114,22 @@ def last_seq(self):
     @property
     def cleanup_complete(self):
         return self._migration_model.cleanup_complete
+
+    def docs_are_replicating(self):
+        """Make a change in source_db and see if it appears in target_db."""
+        source_db, target_db = self.source_db, self.target_db
+        doc_id = "testing-replication-doc"
+        test_doc = {"doc_type": self.doc_types[0],
+                    "_id": doc_id,
+                    "description": self.docs_are_replicating.__doc__}
+        source_db.save_doc(test_doc)
+        sleep(10)
```

Comment:
Only if the process is running on the same machine, and you know what the process is called.  There is probably a cleverer way to figure this out, but I went with this heavy-handed approach because it should catch any kind of replication.